Core/src/IfcGeometryConverter/PlacementConverter.h

```diff
@@ -153,6 +157,22 @@ namespace OpenInfraPlatform {
 					return point * UnitConvert()->getLengthInMeterFactor();
                 }
 
+				/*! \brief Converts \c IfcPointOnCurve to a 3D vector.
+				 *
+				 * \param[in]	pointOnCurve			\c IfcPointOnCurve entity to be interpreted.
+				 *
+				 * \return								Calculated 3D vector.
+				 */
+				carve::geom::vector<3> convertIfcPointOnCurve(
+					const EXPRESSReference<typename IfcEntityTypesT::IfcPointOnCurve>& pointOnCurve
+				) const throw(...) {

```

Comment:
```suggestion
				) const noexpect(false) {
```

We're no longer using `throw(...)` - use this with all *new*  functions you're writting.

Reply:
Resolved: 795b117